Damage.cs

```diff
@@ -4389,6 +4389,48 @@ static Damage()
                     });
 
             Spells.Add(
+                "TahmKench",
+                new List<DamageSpell>
+                    {
+                        //Q
+                        new DamageSpell
+                            {
+                                Slot = SpellSlot.Q, DamageType = DamageType.Magical,
+                                Damage =
+                                    (source, target, level) =>
+                                    new double[] { 80, 125, 170, 215, 260 }[level] 
+                                    + 0.7 * source.FlatMagicDamageMod
+                            },
+                        //W - Devour - Enemy champion
```

Comment:
I think its probably best to check if the target is a minion instead of splitting it into 2 stages.